### ISSUE TITLE
Incorrect data types for createInfiniteQuery

### ISSUE THREAD

**Issue by CoopTRUE** [vishalbalaji/trpc-svelte-query-adapter#25]
`createInfiniteQuery` has the incorrect type for data

For example

```ts
// route.ts
router({
  infinite: protectedProcedure
    .use(logger)
    .input(
      z.object({
        cursor: z.number(),
      })
    )
    .query(async ({ input, ctx }) => {
      console.log(input)
      const { cursor } = input
      const posts = await db.query.clubPosts.findMany({
        limit: limit + 1,
        offset: cursor,
        ...
        orderBy: (post, { desc }) => [desc(post.createdAt)],
      })

      let nextCursor: number | null = null
      if (posts.length > limit) {
        posts.pop()
        nextCursor = cursor + limit
      }
      return { posts, nextCursor }
    }),
})
```
```ts
// page.svelte
  const feed = trpc($page).explore.infinite.createInfiniteQuery(
    {},
    {
      initialPageParam: 0,
      getNextPageParam: ({ nextCursor }) => nextCursor,
    }
  )
```

`$feed.data` incorrectly shows the raw query return type, rather than `pageParams` and `pages`
<img width="290" alt="image" src="https://github.com/vishalbalaji/trpc-svelte-query-adapter/assets/62957956/f3b94c9f-e6b1-4fe1-9bbf-9c2595291a0e">
<img width="556" alt="image" src="https://github.com/vishalbalaji/trpc-svelte-query-adapter/assets/62957956/2f0da464-1b2a-495a-bf29-a1b62b77cf9e">

This can be fixed by wrapping `InfiniteData` on `TData` in the `d.ts`
<img width="765" alt="image" src="https://github.com/vishalbalaji/trpc-svelte-query-adapter/assets/62957956/5cba9cb0-8e53-413e-9395-0f577b06ff15">

**Comment by CoopTRUE** [comment]
Just realized this was fixed in a recent branch
https://github.com/vishalbalaji/trpc-svelte-query-adapter/tree/streamed-queries

Please pull asap

**Comment by vishalbalaji** [comment]
@CoopTRUE This has been merged and should be fixed in v2.2.3. Thanks for raising the issue!
